src/main/views/waveform.zoomview.js

```diff
@@ -80,7 +80,10 @@ define([
 
           // enable drag if necessary
           var mouseMove = function(event) {
-            peaks.seeking = false;
+			var pixelsDifference = Math.abs(event.layerX - x);
```

Comment:
After further testing it is possible to have sub pixel values with css but not with JS move events, they are full integers for now.